python/pyarrow/tests/test_json.py

```diff
@@ -67,6 +76,8 @@ def test_read_options():
     assert opts.block_size == 1234
     assert opts.use_threads is False
 
+    check_options_class_pickling(cls, block_size=1234,
+                                 use_threads=False)
```

Comment:
```suggestion
                                 use_threads=False)

```

The linting error is about this missing blank line